### PR TITLE
[receiver/simpleprometheus] mark as unmaintained

### PR DESCRIPTION
**Pull request by codeboten**
The following PR identifies the simpleprometheusreceiver as unmaintained and logs a warning on start.

### CODE REVIEW

File: receiver/simpleprometheusreceiver/factory.go [hunk 1]
```diff
@@ -16,12 +16,14 @@ package simpleprometheusreceiver // import "github.com/open-telemetry/openteleme
...
 import (
 	"context"
+	"sync"
 	"time"
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/config/confignet"
 	"go.opentelemetry.io/collector/consumer"
+	"go.uber.org/zap"
```
Comment (mx-psi) [inline receiver/simpleprometheusreceiver/factory.go:26]: Needs an update to `go.mod` to be able to use this

File: receiver/simpleprometheusreceiver/factory.go [hunk 2]
```diff
@@ -54,12 +58,19 @@ func createDefaultConfig() config.Receiver {
 	}
 }
 
+func logStatus(logger *zap.Logger) {
+	once.Do(func() {
+		logger.Warn("prometheus_simple receiver is unmaintained and actively looking for contributors.")
```
[the remark this answers is not in the document]
Comment (codeboten) [reply receiver/simpleprometheusreceiver/factory.go:63]: Yes, I wanted to start with something here, but I think we should make this more generic to also include deprecation status. I'll create an issue in the core repo

File: receiver/simpleprometheusreceiver/factory.go (hunk 1 above)
Comment (codeboten) [reply receiver/simpleprometheusreceiver/factory.go:26]: Done

File: receiver/simpleprometheusreceiver/factory.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (codeboten) [reply receiver/simpleprometheusreceiver/factory.go:63]: https://github.com/open-telemetry/opentelemetry-collector/issues/5559

**Review by TylerHelmuth — APPROVED** [review]

**Review by gouthamve — APPROVED** [review]
LGTM

**Review by mx-psi — APPROVED** [review]

**Comment by mx-psi** [comment]
> I don't see any open issues submitted for this component. Is this component marked as unmaintained because it doesn't have a code owner?

I believe that's the case, yes

**Comment by codeboten** [comment]
@dmitryax i've opened an issue to track the component as unmaintained https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1150, in it i also identified the open issues i could find that were related to the component.

**Comment by dmitryax** [comment]
@codeboten looks like the issue link is wrong

**Pull request merged**